src/CyBy/Draw/Internal/Settings.idr

```diff
@@ -4,10 +4,19 @@ import CyBy.Draw.Internal.Abbreviations
 import CyBy.Draw.Internal.Color
 import CyBy.Draw.Internal.CoreDims
 import Text.SVG
+import Derive.Prelude
 import Chem
 import Geom
 
 %default total
+%language ElabReflection
+
+
+||| A data type for the different extensions supported at the moment.
+public export
+data Extension = Word
```

Comment:
Instead of using `Maybe Extension` in the draw state, I suggest to add a default extension such as `None` to this data type.